ChildMindInstitute/mindlogger-app
# 0.14.14 - Fix app icon badge

Fixes the app icon badge 

Issue #623

**Testing this PR**
1. Create several notifications for an applet in the iOS app
2. Check that the app icon badge reflects the number of notifications
3. Open one notification
4. Go back to the home screen and check the app icon badge

**Expected result**
The badge number should have decreased and reflect the current number of notifications
